sql/ttc-bot.sql

```diff
@@ -63,4 +63,11 @@ CREATE TABLE ttc_webhooks(
 	channel_id BIGINT NOT NULL UNIQUE,
 	webhook_url VARCHAR NOT NULL,
 	PRIMARY KEY(channel_id)
-);
\ No newline at end of file
+);
+
+DROP TABLE IF EXISTS ttc_easter_egg_gifs;
+CREATE TABLE ttc_easter_egg_gifs(
+	id SERIAL,
+	content VARCHAR(240) NOT NULL,
```

Comment:
The character limit could be problematic with longer responses, as it is not purely a link to the GIF. As this is not something an end user would be able to write stuff to anyway the length limitation is probably not needed.